### DISCUSSION TITLE
Text boxes are recognized as password fields (Android)

### DISCUSSION THREAD

**Discussion by Jan-Arbus** [ppy/osu#17232] [Q&A] [answered]
My phone is Huawei P40
Android version is 10 
System version is 2.0.0.223(C00E215R3P6)
I can't enter non-English text

https://user-images.githubusercontent.com/74862110/158077613-3473c69f-8bb2-4380-9bd5-c6d035b89f2f.mp4

**Comment by peppy** [comment]
Can you elaborate on why you think they are being recognised as password fields? It's quite possibly the case, but curious where you're seeing the in the provided video/flow.

**Reply by honguyenminh** [reply]
GBoard will show suggestions along with additional features and setting on the top bar, and enable the auto-complete feature shown by the underscore under the current word if it's a normal textbox. I think he suspect that to be a password field because GBoard is not suggesting anything, no word is underlined and all top bar features are disabled, and replaced by the number bar. That happens with password fields on GBoard.

**Reply by Coppertine** [reply]
As well, if using a password manager on your phone, the password manager would ask if you would like to use a saved password for the application. But that might need some testing to ensure that's correct.

**Reply by Jan-Arbus** [reply]
When I click on the textbox, the default popup is secure keyboard
Usually only appears in the password section
So I think the textbox is recognized as a password segment
After enabling secure keyboard, screen recording refuses to work while typing, I can't provide video Sorry

**Comment by grayawa** [comment]
So what's the progress?This bug is still bothering me.

**Comment by Jan-Arbus** [comment, marked as answer]
Now this problem has been solved because of the migration to sdl3
